src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -0,0 +1,129 @@
+    skeliingtonCase.setArid(collectionCase.getAddress().getArid());
+    skeliingtonCase.setLatitude(collectionCase.getAddress().getLatitude());
+    skeliingtonCase.setLongitude(collectionCase.getAddress().getLongitude());
+    skeliingtonCase.setUprn(collectionCase.getAddress().getUprn());
+    skeliingtonCase.setRegion(collectionCase.getAddress().getRegion());
+    skeliingtonCase.setActionPlanId(collectionCase.getActionPlanId()); // This is essential
+    skeliingtonCase.setTreatmentCode(collectionCase.getTreatmentCode()); // This is essential
+    skeliingtonCase.setAddressLevel(collectionCase.getAddress().getAddressLevel());
+    skeliingtonCase.setAbpCode(collectionCase.getAddress().getApbCode());
+    skeliingtonCase.setCaseType(collectionCase.getCaseType());
+    skeliingtonCase.setAddressType(collectionCase.getAddress().getAddressType());
+    skeliingtonCase.setUprn(collectionCase.getAddress().getUprn());
+    skeliingtonCase.setEstabArid(collectionCase.getAddress().getEstabArid());
+    skeliingtonCase.setEstabType(collectionCase.getAddress().getEstabType());
+    skeliingtonCase.setOrganisationName(collectionCase.getAddress().getOrganisationName());
+    skeliingtonCase.setOa(collectionCase.getOa());
+    skeliingtonCase.setLsoa(collectionCase.getLsoa());
+    skeliingtonCase.setMsoa(collectionCase.getMsoa());
+    skeliingtonCase.setLad(collectionCase.getLad());
+    skeliingtonCase.setHtcWillingness(collectionCase.getHtcWillingness());
+    skeliingtonCase.setHtcDigital(collectionCase.getHtcDigital());
+    skeliingtonCase.setFieldCoordinatorId(collectionCase.getFieldCoordinatorId());
+    skeliingtonCase.setFieldOfficerId(collectionCase.getFieldOfficerId());
+    skeliingtonCase.setCeExpectedCapacity(collectionCase.getCeExpectedCapacity());
+    skeliingtonCase.setCeActualResponses(collectionCase.getCeActualResponses());
+    skeliingtonCase.setHandDelivery(collectionCase.isHandDelivery());
+
+    skeliingtonCase.setSurvey("CENSUS");
+    skeliingtonCase.setRefusalReceived(false);
+    skeliingtonCase.setReceiptReceived(false);
+    skeliingtonCase.setAddressInvalid(false);
+    skeliingtonCase.setUndeliveredAsAddressed(false);
+    return skeliingtonCase;
+  }
+
+  // https://collaborate2.ons.gov.uk/confluence/display/SDC/Handle+New+Address+Reported+Events
+  // Only a small number of mandatory fields to create a skellington case
+  private UUID validDateNewAddressCollectionCaseForMandatoryFields(
+      CollectionCase newCollectionCase) {
+    UUID newCaseId;
+
+    try {
+      newCaseId = UUID.fromString(newCollectionCase.getId());
+    } catch (IllegalArgumentException e) {
+      throw new RuntimeException(
+          "Expected NewAddress CollectionCase Id to be a valid UUID, got: "
+              + newCollectionCase.getId());
+    }
+
+    List<String> valiidCaseTypes = Arrays.asList(new String[] {"HH", "HI", "CE", "SPG"});
+
+    if (!valiidCaseTypes.contains(newCollectionCase.getCaseType())) {
```

Comment:
why are we doing validation like this? The tasks say "validate for minumum contents" (i.e. mandatory fields) not this. This is just more code to maintain.